commit 51d636df06aac4cae32ada625f0f1bb6dda4821d
Author: John Helmert III <ajak@gentoo.org>
Date:   Tue Jan 17 09:39:32 2023 -0600

    output_test: skip test broken on dates other than 2023-01-07

    Confusingly, doesn't seem reproducible from a local checkout.
     
    Signed-off-by: John Helmert III <ajak@gentoo.org>

diff --git a/internal/audit/output_test.go b/internal/audit/output_test.go
index 027aa3a5..34ef5387 100644
--- a/internal/audit/output_test.go
+++ b/internal/audit/output_test.go
@@ -10,6 +10,8 @@ import (
 )
 
 func TestHTML(t *testing.T) {
+	t.Skip("Broken on dates other than 2023-01-07")
+
 	r := newReport()
 
 	r.AddPassword("foo", "bar")

commit 452fac08c13a7792fdad2eae404f27bd668b2fb1
Author: John Helmert III <ajak@gentoo.org>
Date:   Tue Jan 17 09:39:13 2023 -0600

    clone_test: skip test which tries to run gpg-agent
    
    Signed-off-by: John Helmert III <ajak@gentoo.org>

diff --git a/internal/action/clone_test.go b/internal/action/clone_test.go
index c2757207..a5c4a21b 100644
--- a/internal/action/clone_test.go
+++ b/internal/action/clone_test.go
@@ -141,6 +141,8 @@ func TestCloneGetGitConfig(t *testing.T) {
 }
 
 func TestCloneCheckDecryptionKeys(t *testing.T) {
+	t.Skip("TODO: Trying to use gpg-agent fails in Portage tests")
+
 	u := gptest.NewUnitTester(t)
 
 	buf := &bytes.Buffer{}
